themes/material/core/loginuserpass.php

```diff
@@ -44,7 +44,7 @@ function onRecaptchaLoad() {
     <main class="mdl-layout__content" layout-children="column" child-spacing="center">
         <?php include __DIR__ . '/../common-announcement.php' ?>
 
-        <form method="post" autocomplete="off" onsubmit="event.target.onsubmit = event => event.preventDefault()">
+        <form method="post" autocomplete="off" onsubmit="function changeSubmit(event) { event.target.onsubmit =  pd(event); function pd(e) { e.preventDefault() }}">
```

Comment:
This doesn't quite look right to me...

Previously, the code in `onsubmit` would set the given event's form's onsubmit to a function that would simply call preventDefault() on the event it was given, meaning that subsequent submissions wouldn't do anything.

Now, the `onsubmit` is simply defining a function that, if it was called, would set the event's form's onsubmit to the **result** of calling a function that calls preventDefault() on the original form submission's event.

I think this line would probably need to be something like...
```suggestion
        <form method="post" autocomplete="off" onsubmit="event.target.onsubmit = preventDefault">
```
... and some new functions would need to be added to a script tag somewhere (such as the one in the `<head>`) like this:
```js
function preventDefault(event) {
  event.preventDefault();
}
```

Reply:
yeah, that's better, make it so